modules/book.js

```diff
@@ -0,0 +1,39 @@
+const bookList = document.querySelector('.book-list');
+
+export default class Book {
+  constructor(title, author, id) {
+    this.title = title;
+    this.author = author;
+    this.id = id;
+  }
+
+  addBook = () => {
+    Book.books.push(this);
+    localStorage.setItem('bookInfo', JSON.stringify(Book.books));
+    Book.displayUI();
+  }
+
+  removeBook = () => {
+    const removeBookID = parseInt(this.id, 10);
+    let listSection = Book.books;
+    listSection = listSection.filter((element, index) => index !== removeBookID);
+    Book.books = listSection;
+    localStorage.setItem('bookInfo', JSON.stringify(listSection));
+    window.location.reload();
+  }
+
+  static displayUI() {
+    let bookDIV = '';
+    Book.books = JSON.parse(localStorage.getItem('bookInfo'));
+    if (Book.books === null) {
+      Book.books = [];
+    }
```

Comment:
- As the code division is super important for making the code cleaner, more readable, and optimized. Please consider moving the `displayUI` function to another file in the `modules` dir in order to make the logic which is adding and removing a book in a file, and the UI in a separate file.